File: service/SurfWebAutofac/Model/Dtos/Rankings/RankingDto.cs
using Model.Models.Main;

namespace Model.Dtos.Rankings;

public class RankingDto
{
    /// <summary>
    ///     排名类型(积分)
    /// </summary>
    public RankingTypeEnum Type { get; set; }

    /// <summary>
    ///     玩家id
    /// </summary>
    public long PlayerId { get; set; }

    /// <summary>
    ///     玩家名
    /// </summary>
    public string PlayerName { get; set; }

    /// <summary>
    ///     数值
    /// </summary>
    public decimal Value { get; set; }

    /// <summary>
    ///     进度
    /// </summary>
    public string Progress { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/rankings/RankingDto.java
package com.surfweb.system.dto.rankings;

import com.surfweb.common.enums.RankingTypeEnum;
import lombok.Data;

import java.math.BigDecimal;

@Data
public class RankingDto {

    /**
     * 排名类型(积分)
     */
    private RankingTypeEnum type;

    /**
     * 玩家id
     */
    private long playerId;

    /**
     * 玩家名
     */
    private String playerName;

    /**
     * 数值
     */
    private BigDecimal value;

    /**
     * 进度
     */
    private String progress;
}
